client/src/test/java/model/ShipMapperTest.java

```diff
@@ -16,7 +16,7 @@ public void givenMockedShipWhenMapToShipModelThenReturnCorrectShipModel() {
     int expectedNumberOfMasts = 2;
     List<Integer> expectedFields = Arrays.asList(0, 1);
     Ship shipToMap = mock(Ship.class);
```

Comment:
it's better to use a real ship instead of a mock.

Also, test name could be simplified, b/c it is very straightforward. It's harder to read it than to understood what is happening there :)

Reply:
This is not my code, this will be covered by Emilia in next pull request